orttraining/orttraining/core/graph/training_op_defs.cc

```diff
@@ -504,32 +504,30 @@ void RegisterTrainingOpSchemas() {
           "Constrain input and output types to numeric tensors.")
       .FunctionBody([]() {
         auto nodes = ONNX_NAMESPACE::FunctionBodyHelper::BuildNodes(
-          {// nodes: {outputs, op, inputs, attributes}
-
-           // Get input shapes and dynamic reduction axes.
-           {{"shape_A"}, "Shape", {"A"}},
-           {{"shape_B"}, "Shape", {"B"}},
-           {{"axes_A", "axes_B"}, "BroadcastGradientArgs", {"shape_A", "shape_B"}},
-
-           // dA = reshape(reduce_sum(dY / B, axes_A), shape_A)
-           {{"dY_over_B"}, "Div", {"dY", "B"}},
-           {{"reduce_dA"}, "ReduceSumTraining", {"dY_over_B", "axes_A"},
-            {ONNX_NAMESPACE::MakeAttribute("noop_with_empty_axes", int64_t(1))}},
-           {{"dA"}, "Reshape", {"reduce_dA", "shape_A"}},
-
-           // dB = reshape(reduce_sum(dY * -A / (B * B)), axes_B), shape_B)
-           {{"B_squared"}, "Mul", {"B", "B"}},
-           {{"minus_A"}, "Neg", {"A"}},
-           {{"minus_A_over_B_squared"}, "Div", {"minus_A", "B_squared"}},
-           {{"pre_reduce_dB"}, "Mul", {"dY", "minus_A_over_B_squared"}},
-           {{"reduce_dB"}, "ReduceSumTraining", {"pre_reduce_dB", "axes_B"},
-            {ONNX_NAMESPACE::MakeAttribute("noop_with_empty_axes", int64_t(1))}},
-           {{"dB"}, "Reshape", {"reduce_dB", "shape_B"}}});
-
-           for (size_t contrib_node_index : {2, 4, 10}) {
-             nodes[contrib_node_index].set_domain(kMSDomain);
-           }
-           return nodes;
+            {// nodes: {outputs, op, inputs, attributes}
+
+             // Get input shapes and dynamic reduction axes.
+             {{"shape_A"}, "Shape", {"A"}},
+             {{"shape_B"}, "Shape", {"B"}},
+             {{"axes_A", "axes_B"}, "BroadcastGradientArgs", {"shape_A", "shape_B"}},
```

Comment:
the changes in this file are not described in your PR, is this fixing another issue?